jmclient/jmclient/wallet.py

```diff
@@ -143,28 +150,46 @@ def _load_storage(self):
                 index = int(utxo[self.TXID_LEN:])
                 md_data[(txid, index)] = value
 
+        # Wallets may not have any metadata
+        if self.METADATA_KEY in self.storage.data.keys():
```

Comment:
redundant `.keys()`

Reply:
done in 1925e0da95a1a1180431a4c3e89a35195f55d889